src/com/company/Main.java

```diff
@@ -51,14 +39,32 @@ private static void printMessage() {
         String memberText = member.nextLine();
     }
 
-    private static void printName(){
+    private static void printName(ArrayList<String> listOfChatMembers, int activeMembersCount) {
         Random rand = new Random();
-        int min = 1;
-        int max = 5;
-        int random = rand.nextInt(max-min+1) + min;
-        System.out.print("Member"+random+ " name:");
-    }
+        String randomString = listOfChatMembers.get(rand.nextInt(listOfChatMembers.size()));
+        System.out.print(randomString);
+
+        int actionNumber = 0;
+        do {
+            Scanner action = new Scanner(System.in);
+            actionNumber = action.nextInt();
+        }
+        while (actionNumber != 1 && actionNumber != 2);
 
+        if (actionNumber == 1) {
+            printMessage();
+        }
+        if (!(listOfChatMembers.isEmpty()) && actionNumber == 2) {
```

Comment:
no need to check listOfChatMembers.isEmpty, as it can't be empty when you get randomString from it